### PR TITLE
Update dependencies for webpack role

### PR DESCRIPTION
**Pull request by simonbru**
Current version of the `mini-css-extract-plugin` loader supports hot module reloading, so we don't need `style-loader` anymore.

There are breaking changes for `css-loader` and `file-loader` but they shouldn't impact our configuration
https://github.com/webpack-contrib/css-loader/blob/master/CHANGELOG.md#breaking-changes
https://github.com/webpack-contrib/file-loader/blob/master/CHANGELOG.md#400-2019-06-05

* This PR is a : Improvement

- [x] Documentation is written
- [x] [Changelog](https://github.com/liip/drifter/blob/master/CHANGELOG.md) was updated

### CODE REVIEW

File: provisioning/roles/nodejs/templates/package.json.webpack.j2 [hunk 1]
```diff
@@ -9,26 +9,25 @@
...
-    "file-loader": "^2.0.0",
-    "mini-css-extract-plugin": "^0.4.0",
-    "node-sass": "^4.8.3",
-    "postcss": "^7.0.5",
-    "postcss-cli": "^6.0.0",
+    "@babel/core": "^7.4.5",
+    "@babel/preset-env": "^7.4.5",
+    "autoprefixer": "^9.6.0",
+    "babel-loader": "^8.0.6",
+    "css-loader": "^2.1.1",
```
Comment (LeBenLeBen) [inline provisioning/roles/nodejs/templates/package.json.webpack.j2]: I see a major bump here, we should ensure our configuration/usage is still adapted.

Comment (simonbru) [reply provisioning/roles/nodejs/templates/package.json.webpack.j2]: Yeah, there is a link to the changes in my first comment. They changed the way imports are handled, but in our case `sass-loader` already handles imports before `css-loader` (as far as I understand).

However I could have missed something, this is worth another look.

Comment (LeBenLeBen) [reply provisioning/roles/nodejs/templates/package.json.webpack.j2]: Oh sorry, I missed your initial statement, it indeed looks good. I'll try to setup it once from scratch just to be sure.

Comment (sephii) [reply provisioning/roles/nodejs/templates/package.json.webpack.j2]: I tried it on a project and it worked just fine, I think we can merge this.

Comment (LeBenLeBen) [reply provisioning/roles/nodejs/templates/package.json.webpack.j2]: Sure then, go ahead!

**Pull request merged**